ui/src/turing/components/configuration/StandardEnsemblerConfigDetails.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { ConfigProvider, useConfig } from "config";
+import { EuiFlexGroup, EuiFlexItem } from "@elastic/eui";
+import { ConfigSectionPanel } from "components/config_section/ConfigSectionPanel";
+import { LinkedRoutesTable } from "../form/standard_ensembler/LinkedRoutesTable";
```

Comment:
Let's use absolute path for all imports.

Reply:
Should I keep the relative paths just for imports within the same folder (`./**`)? It seems like we are sometimes keeping the relative paths (to supposedly better reflect the hierarchy of the imported components) and other times preferring absolute paths 👀

Reply:
I remember we discussed this a while ago on a different PR. I don't think there is a strong convention around it. In general, I think the absolute imports are convenient to work with especially for importing shared components (you don't have to think about where a component is located in relation to the current component). So I'd definitely avoid `../`.

But when we are importing a component that's nested within the current component's structure, I think relative imports are ok - in fact, you can make the reverse argument now (easier to do `./EnsemblerDetailsRow` than `turing/components/configuration/StandardEnsembler/EnsemblerDetailsRow` or something like that). The rule of thumb I'm trying to use recently is - do the 2 components have a composition kind of relationship, where you can always say that one will only be used by the other (=> if the parent moves, the child component will just move with it)? If so, I think `./` is ok (and may be preferable). `../` is a strict no. 😅